go.mod

```diff
@@ -1,16 +1,18 @@
 module github.com/container-storage-interface/spec
 
-go 1.18
+go 1.23.0
+
+toolchain go1.24.4
 
 require (
-	google.golang.org/grpc v1.57.1
-	google.golang.org/protobuf v1.33.0
+	github.com/golang/protobuf v1.5.4
```

Comment:
How did this deprecated link become a direct reference?  It should be an indirect one used by google.golang.org/protobuf internally.